shp:

    # KAIST
    # lane: "/HD_MAP/HD_Map_KAIST_V2/2_LANE_SIDE_KAIST_V2/2_LANE_SIDE_KAIST_V2.shp"
    # link: "/HD_MAP/HD_Map_KAIST_V2/1_LANE_LINK_KAIST_V2/1_LANE_LINK_KAIST_V3.shp"
    # node: "/HD_MAP/HD_Map_KAIST_V2/6_NODES_KAIST_V2/6_NODES_KAIST_V2.shp"

    # # ALPHA ( 대구 수성의료지구 )
    # lane: "/HD_MAP/HD_MAP_alpha_atypical_out/LANE/A1_LANE_densifiedV3.shp"
    # link: "/HD_MAP/HD_MAP_alpha_atypical_out/LINK/A3_LINK_reverse_densifiedV6.shp"
    # node: "/HD_MAP/HD_MAP_alpha_atypical_out/NODE/C1_NODEV5.shp"

    # ALPHA ( 대구 수성의료지구 )
    lane: "/HD_MAP/KIAPI_HD_MAP_bumyong/LANE.shp"
    link: "/HD_MAP/KIAPI_HD_MAP_bumyong/LINK.shp"
    node: "/HD_MAP/KIAPI_HD_MAP_bumyong/NODE.shp"

    # # KAIST
    # lane: "/HD_MAP/HD_Map_KAIST_LINK_LANE/2_LANE_SIDE_KAIST_V2/2_LANE_SIDE_KAIST_V2.shp"
    # link: "/HD_MAP/HD_Map_KAIST_LINK_LANE/1_LANE_LINK_KAIST_V2/1_LANE_LINK_KAIST_V2.shp"
    # node: "/HD_MAP/HD_Map_KAIST_LINK_LANE/6_NODES_KAIST_V2/6_NODES_KAIST_V2.shp"
layer:
    # # KAIST
    # lane: 2_LANE_SIDE_KAIST_V2
    # link: 1_LANE_LINK_KAIST_V2
    # node: 6_NODES_KAIST_V2

    # # ALPHA ( 대구 수성의료지구 )
    # lane: A1_LANE_densifiedV3
    # link: A3_LINK_reverse_densifiedV6
    # node: C1_NODEV5

    # ALPHA ( 대구 수성의료지구 )
    lane: LANE
    link: LINK
    node: NODE

# path:
#     start: 155M00500204
#     goal: 155M02240104

# PG
lane:
    id: 9
    right_link: 0
    left_link: 1
    lanetype: 2
    lanecode: 3
    barrier: 4
    laneno: 5
    code: 6

# PG
link:
    id: 0
    from_node: 1
    to_node: 2
    length: 3
    speed: 6
    left_link: 16
    right_link: 17
    next_link: 18


# PG
node:
    id: 0
    next_link: 8
    prev_link: 9

rviz_on:
    is_link_on: true
    is_node_on: true
    is_lane_on: true
    is_path_on: true
